ui/utils_test.py

```diff
@@ -1,11 +1,21 @@
 """Tests for utils methods"""
 from tempfile import NamedTemporaryFile
+import json
+
 import pytest
 from zeep.exceptions import Fault
 
 from ui import factories
-from ui.exceptions import MoiraException
-from ui.utils import write_to_file, get_moira_client, user_moira_lists, has_common_lists
+from ui.exceptions import MoiraException, GoogleAnalyticsException
+from ui.utils import (
+    write_to_file, get_moira_client, user_moira_lists,
```

Comment:
very minor thing, feel free to ignore here if you prefer: if most methods are on separate lines, do them all that way for consistency:
```python
from ui.utils import (
    write_to_file, 
    get_moira_client, 
    user_moira_lists,
```

Reply:
sounds good, updated to put methods on separate lines